default.nix

```diff
@@ -8,7 +8,11 @@ let
   vessel = rustPlatform.buildRustPackage rec {
     pname = "vessel";
     version = "0.6.5";
-    buildInputs = [ openssl_3_0 openssl_3_0.dev ];
+    buildInputs = [ 
+      openssl_3_0 openssl_3_0.dev 
+      ] ++ pkgs.lib.optionals pkgs.stdenv.isDarwin [
+        pkgs.darwin.apple_sdk.frameworks.Security
+      ];
```

Comment:
```suggestion
      ] ++ pkgs.lib.optional pkgs.stdenv.isDarwin
        pkgs.darwin.apple_sdk.frameworks.Security;
```

Reply:
Thanks for this @luc-blaeser! If MacOS failed in the past, that implies we didn't have a CI job for it?